packages/electron/lib/install.js

```diff
@@ -55,6 +56,7 @@ module.exports = {
   },
 
   checkIconVersion () {
+    // TODO: this seems wrong, it's hard coding the check only for OSX and not windows or linux (!?)
```

Comment:
@flotwig might want to follow up with @elevatebart who was the original author of this code below. it seems completely wrong and I'm not sure what would even happen on `linux` or `win32`. My guess is that it would **always** rebuild the electron binary since this check would presumably fail every single time. If that's the case then we're rebuilding the electron binary unnecessarily every single time

Reply:
Hm, this is definitely incorrect, but I think it's actually doing the opposite - it's not rebuilding on linux/win32 if the icons change, only on macOS. If I run `yarn build` in `packages/electron` I get no output about this error being thrown or electron reinstalling. Given that we very rarely change icons and this is a dev-only issue, I'm ok with leaving this TODO as a note without creating an issue.